tools/v8_gypfiles/v8.gyp

```diff
@@ -468,17 +460,6 @@
                 'mksnapshot_flags': ['--v8_os_page_size', '<(v8_os_page_size)'],
               },
             }],
-            ['v8_use_external_startup_data', {
-              'outputs': ['<(INTERMEDIATE_DIR)/snapshot_blob.bin', ],
-              'variables': {
-                'mksnapshot_flags': ['--startup_blob', '<(INTERMEDIATE_DIR)/snapshot_blob.bin', ],
-              },
-            }, {
-               'outputs': ["<(INTERMEDIATE_DIR)/snapshot.cc"],
-               'variables': {
-                 'mksnapshot_flags': ['--startup_src', '<(INTERMEDIATE_DIR)/snapshot.cc', ],
-               },
-             }],
```

Comment:
You need to keep the else block.

Reply:
Yes! I moved this up since this isn't really conditional anymore.